### PR TITLE
Test for disable/enable namespace

### PR DESCRIPTION
**Pull request by harish-24**
Add test to verify disabling/enabling namespace of a region

Signed-off-by: Harish <harish@linux.ibm.com>

### CODE REVIEW

**Comment by harish-24** [comment]
```
# avocado run memory/ndctl.py:test_disable_enable_ns
...
 (1/1) memory/ndctl.py:NdctlTest.test_disable_enable_ns: FAIL: Namespace enable failed for "all" (15.31 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 15.37 s
```

File: memory/ndctl.py [hunk 1]
```diff
@@ -173,12 +173,29 @@ def disable_namespace(self, namespace='all', region='', bus='',
             args = '%s -r %s' % (args, region)
         if bus:
             args = '%s -b %s' % (args, bus)
-        if force:
-            args = '%s -f' % args
+        if verbose:
```
Comment (PraveenPenguin) [inline memory/ndctl.py]: any specfic reason to move force to verbose please have that part to commit mesage

File: memory/ndctl.py [hunk 2]
```diff
@@ -322,6 +339,23 @@ def test_namespace(self):
         namespaces = self.get_json('-N')
         self.log.info('Created namespace %s', namespaces)
 
+    def test_disable_enable_ns(self):
+        """
+        Test enable disable namespace
+        """
+        region = self.get_default_region()
+        for _ in range(0, 3):
```
Comment (PraveenPenguin) [inline memory/ndctl.py]: why loop ?

File: memory/ndctl.py (hunk 1 above)
Comment (harish-24) [reply memory/ndctl.py]: That is the correction we are doing here as it does not have a force option. Will have that as part of commit message

File: memory/ndctl.py (hunk 2 above)
Comment (harish-24) [reply memory/ndctl.py]: to execute multiple times

Comment (PraveenPenguin) [reply memory/ndctl.py]: :) yes i understand that but the question is test requires to create three namespaces ?

File: memory/ndctl.py (hunk 1 above)
Comment (PraveenPenguin) [reply memory/ndctl.py]: :+1:

File: memory/ndctl.py (hunk 2 above)
Comment (harish-24) [reply memory/ndctl.py]: yes to enable/disable namespaces it has to be created first :)

**Comment by harish-24** [comment]
@PraveenPenguin Pushed with commit message changes. Thanks.

**Pull request merged**